modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -941,6 +941,7 @@ export default class AvgRTPStatsReporter {
             return;
         }
         const transportStats = {
+            isP2P: data.transport[0].p2p,
```

Comment:
can we keep the the p2p flag named `p2p` like other parts of the stats  report?

Reply:
LGTM otherwise

Reply:
ok sure